packages/cozy-stack-client/src/SharingCollection.js

```diff
@@ -231,6 +234,8 @@ export const getSharingRules = (document, sharingType) => {
       `sharingType is deprecated and will be removed. We now set this default rules: ${
         isFile(document)
           ? getSharingRulesForFile(document)
+          : document._type === BITWARDEN_ORGANIZATIONS_DOCTYPE
+          ? getSharingRulesForOrganizations(document)
           : getSharingRulesForPhotosAlbum(document)
```

Comment:
Here the linter doesn't let me to indent the nested ternary operator.

I find this version easier to read :
```js
isFile(document)
  ? getSharingRulesForFile(document)
  : document._type === BITWARDEN_ORGANIZATIONS_DOCTYPE
    ? getSharingRulesForOrganizations(document)
    : getSharingRulesForPhotosAlbum(document)
```

Please tell me what do you prefer :
- to keep current format
- to find a way to indent it correctly -> need to edit linter configuration, maybe to upgrade its version
- rewrite the code with classic if/else code

Reply:
Le 2 idéalement, sans y passer des heures. Si le 1 t'as posé problème à la lecture, go pour le 3 ?

Reply:
Personally, I would go for the 3rd. I don't like nested ternary

Reply:
I finally chose to rewrite it to the old fashion way. Current linter version seems not to handle ternary operator indentation and upgrading needs some edits on the project's configuration.

https://github.com/cozy/cozy-client/pull/990/files#diff-eedaf5c6b0118a4e2a44524e49bf386befb317815ee159d97a2242a578b96724R229-R240